.github/workflows/ci.yml

```
@@ -226,22 +226,36 @@ jobs:
 
   ecosystem-test-prefect:
     needs: build-binary-linux
-    name: "ecosystem test | prefect"
+    name: "ecosystem test | ${{ matrix.repo }}"
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - {
+              repo: "prefecthq/prefect",
+              command: "uv pip install --upgrade -e '.[dev]'",
+              python: "3.9",
+            }
+          - {
+              repo: "home-assistant/core",
+              command: "uv pip install -r requirements_all.txt",
+              python: "3.12",
+            }
```

I think we can skip the parentheses

```suggestion
          - repo: "prefecthq/prefect",
            command: "uv pip install --upgrade -e '.[dev]'",
            python: "3.9",
          - repo: "home-assistant/core",
            command: "uv pip install -r requirements_all.txt",
            python: "3.12",
```